cms/static/js/models/settings/course_details.js

```diff
@@ -62,6 +67,15 @@ var CourseDetails = Backbone.Model.extend({
         // NOTE don't return empty errors as that will be interpreted as an error state
     },
 
+    convertDateStringsToObjects: function(attrs){
+        dateFields = ["start_date", "end_date", "enrollment_start", "enrollment_end"];
```

Comment:
We can pass these fields into the newly-moved utility method as a function argument